### ISSUE TITLE
Anaconda tensorflow-gpu faulty package?

### ISSUE THREAD

**Issue by OverLordGoldDragon** [tensorflow/tensorflow#33795]
Using [`conda install -c anaconda tensorflow-gpu`](https://anaconda.org/anaconda/tensorflow-gpu) yields the pre-installation message below, showing that cuDNN 7.6.0 will be installed _against_ the TensorFlow GPU [compatibility table](https://www.tensorflow.org/install/source_windows#gpu). `pip` install does _not_ include cuDNN. Further, I already have cuDNN 7.4.2 installed per [official instructions](https://docs.nvidia.com/deeplearning/sdk/cudnn-install/index.html#install-windows) with paths [set up](https://www.tensorflow.org/install/gpu#windows_setup).

Either the compatibility table is outdated, or the Anaconda package is faulty - which is it?

<hr>

<img src="https://user-images.githubusercontent.com/16495490/67711092-ae104780-f9da-11e9-8651-dad5ff602319.png" width="550">

<img src="https://user-images.githubusercontent.com/16495490/67711269-13fccf00-f9db-11e9-9f13-53e5fab60bfc.png" width="650">

**Comment by OverLordGoldDragon** [comment]
**UPDATE**: proceeded w/ Conda install, it worked - but only after installing `keras_applications` and `keras_preprocessing` via `pip`. Thus, the compatibility table appears outdated. 

I remain unsure how Conda's 7.6 install doesn't conflict with my manual 7.4 install, however.

**Comment by ymodak** [comment]
The compatibility table tells us about the configurations which are tested and were able to build TF successfully. However it does not cover all successful build configurations but only a subset of it.
So the tested build configurations table is not outdated.
I will close this issue now since you are able to install TF. Thanks!

**Comment by OverLordGoldDragon** [comment]
@ymodak While I've made an install, I now face a compatibility issue w/ Keras 2.3.1 - latter aside, my question on cuDNN compatibility remains: can cuDNN 7.4.2 installed from source, manually, run with cuDNN 7.6.0 installed by Anaconda (into a conda environment w/ TF2 install)? I can open a separate issue on this if needed.

**Comment by gunan** [comment]
We do not build anaconda packages ourselves.
Therefore, I am not sure.
Anaconda or the community may be building TF with different cuda/cudnn versions than we are doing, so that table there does not apply to what you download from anaconda.
